### PR TITLE
IDEM-252

### PR DESCRIPTION
**Pull request by JasminFarhi**
Image from iOS (7)
Image from iOS (6)

### CODE REVIEW

**Review by xiangzhe0812 — APPROVED** [review]

File: data/sites.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "tagline": "The easiest way to get paid in Bitcoin",
     "description": "",
     "website": "https://getpaidinbitcoin.com.au",
-    "logo": "https://images.squarespace-cdn.com/content/v1/5f5990cc9e792617828214ae/1600921877148-3ISFDDENM1N89GLGAIKI/GBIB+Logo_5%402x.png?format=1500w",
+    "logo": "https://images.squarespace-cdn.com/content/v1/5f5990cc9e792617828214ae/1600921877148-3ISFDDENM1N89GLGAIKI/GBIB+Logo_5%402x.png",
```
Comment (bitcoinbrisbane) [inline data/sites.json:20]: The image needs to be in our repo

**Review by bitcoinbrisbane — CHANGES_REQUESTED** [review]
The image needs to be in our repo

File: data/sites.json (hunk 1 above)
Comment (JasminFarhi) [reply data/sites.json:20]: This feedback does not relate to the ticket. Happy to do this but I think it should be scoped out in another ticket. :) @bitcoinbrisbane

Comment (bitcoinbrisbane) [reply data/sites.json:20]: Ok, other vendor

**Review by bitcoinbrisbane — APPROVED** [review]

**Pull request merged**